std/stdio.d

```diff
@@ -2981,8 +2981,16 @@ $(D Range) that locks the file and allows fast writing to it.
     }
 
     /**
+     * Output range which locks the file when created, and unlocks it when it goes
+     * out of scope.
+     *
      * Returns: An $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
-     * that locks the file and allows fast writing to it.
+     * which accepts string types, `ubyte[]`, individual character types, and
+     * individual `ubyte`s.
+     *
+     * Note: Writing either arrays of `char`s or `ubyte`s is much faster than
+     * writing each character individually a range. For large files,
+     * writing using an array acting as output buffer can result in a speed increase.
```

Comment:
"individually a range" -> ??? Not sure what you meant here, but it needs rewording

"For large files, writing using an array acting as output buffer can result in a speed increase."

Isn't this redundant? You just said writing an array is "much faster". I don't get the point about "acting as an output buffer".